boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w.dts

```diff
@@ -68,6 +69,26 @@ uart2: &scb2 {
 	};
 };
 
+&sdhc0{
+	status = "okay";
+	compatible = "infineon,cat1-sdhc-sdio";
+
+	/* SDIO pins */
+	pinctrl-0 = <&p2_4_sdio_cmd &p2_5_sdio_clk &p2_0_sdio_data0
+				 &p2_1_sdio_data1 &p2_2_sdio_data2 &p2_3_sdio_data3>;
+	pinctrl-names = "default";
+
+	/* Wi-Fi configuration */
+	wifi-sdio {
+		status = "okay";
+		compatible = "infineon,airoc-wifi-sdio";
```

Comment:
we can remove '-sdio' and leave "infineon,airoc-wifi". The "wifi-host-wake-gpios", wifi-host-wake-gpios" will common for SDIO and SPI.